winCompile.bat

```diff
@@ -4,3 +4,5 @@ call npm run compile
 
 echo copying acadProcessFinder.exe...
 copy utils\acadProcessFinder\bin\acadProcessFinder.exe out\process
+echo copying webHelpAbstraction.json...
+copy extension\src\help\webHelpAbstraction.json out\help
```

Comment:
If this feature also wants to support on Mac, maybe we need to change the macCompile.bat